### PR TITLE
added juniper show bgp summary

### PR DESCRIPTION
**Pull request by ysanajong**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
Juniper junos show bgp summary 

##### SUMMARY
Adds parsing for show bgp summary output on Juniper JunOS

### CODE REVIEW

**Comment by ysanajong** [comment]
Can someone review this please?

File: ntc_templates/templates/juniper_junos_show_bgp_summary.textfsm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ^Groups: -> BGP
+  ^\s+. -> Error
+
+BGP
+  ^Peer\s+AS\s+InPkt\s+OutPkt\s+OutQ\s+Flaps\s+Last\s+Up\/Down\s+State\|#Active\/Received\/Accepted\/Damped...$$
+  ^${PEER_IP}\s+${PEER_AS}\s+${IN_PKT}\s+${OUT_PKT}\s+${OUT_Q}\s+${FLAPS}\s+${LAST_FLAP}\s+${UP_DOWN}\s+${STATE} 
+  ^\s+${INTERFACE}:\s+${ACTIVE}\/${RECEIVED}\/${ACCEPTED}\/${DAMPED} -> Record
+  ^\s+. -> Error
+
+EOF
```
Comment (jvanderaa) [inline ntc_templates/templates/juniper_junos_show_bgp_summary.textfsm:26]:
```suggestion

```

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Remove `EOF` that is not present on other templates.

**Comment by jmcgill298** [comment]
replaced by https://github.com/networktocode/ntc-templates/pull/1744

**Pull request closed without merging**